doc/manual/50-feature-details.en.md

```diff
@@ -73,6 +70,52 @@ incremental backup as follows:
 barman backup --reuse-backup=link <server_name>
 ```
 
+> **NOTE:** Unlike Postgres block-level incremental backups, Rsync file-level incremental backups are independent
+> on their own, meaning that a backup that reused a previous backup for deduplication is not compromised
+> in any way if that backup is deleted. As mentioned, deduplication in Rsync backups is implemented with the use
+> of hard links, so when a previously reused backup is deleted, files shared with other backups will
+> still remain on disk, only being removed when the last backup using those files is also deleted. It also
+> means that there’s no need to take new full backups when a previous full backup is deleted, each backup
+> will still have all files and can be reused without any concerns.
+
+> **IMPORTANT:** The `reuse_backup` option must be used along with
+> `rsync` or `local-rsync` as backup method.
+
+
+### Block-level incremental backups
+
+Since version 3.11, Barman introduces support for block-level incremental
+backups, leveraging the native [incremental backup support introduced in PostgreSQL 17](https://www.postgresql.org/docs/17/continuous-archiving.html#BACKUP-INCREMENTAL-BACKUP).
```

Comment:
We can leave this for now, but once 17 is out and is the current version, we should change 17 for current. Let's do that as part of the next release.
This means we need a JIRA ticket so we don't forget ;)